Essentials/src/config.yml

```diff
@@ -709,6 +709,10 @@ max-money: 10000000000000
 # Setting this to 0, will disable overdrafts/loans completely.  Users need 'essentials.eco.loan' perm to go below 0.
 min-money: -10000
 
+# If you don't want buy and sell prices to be the same, set this multiplier to something higher then 1.0
+# All buy prices on buy signs will be the worth value multiplied by this amount.
+economy-buy-multiplier: 1.0
```

Comment:
maybe just "buy-multiplier" since this is already in the economy section

Reply:
makes sense